django_extensions/management/modelviz.py

```diff
@@ -8,6 +8,8 @@
   Adapted to be used with django-extensions
 """
 
+# from __future__ import unicode_literals
```

Comment:
Actually this is commented and will not be executed. Probably it is not needed at all and everything will work regardless.